### PR TITLE
Laura/more score param tuning reduce amplification

### PR DESCRIPTION
**Pull request by laura-makdah**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by laura-makdah** [comment]
@dmitri-perelman this is ready for review, I may remove the last commit depending on private testnet results

File: narwhal/node/src/primary_node.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@ impl PrimaryNodeInner {
     /// The window where the schedule change takes place in consensus. It represents number
     /// of committed sub dags.
     /// TODO: move this to node properties
-    const CONSENSUS_SCHEDULE_CHANGE_SUB_DAGS: u64 = 300;
+    const CONSENSUS_SCHEDULE_CHANGE_SUB_DAGS: u64 = 10_000;
```
Comment (dmitri-perelman) [inline narwhal/node/src/primary_node.rs:52]: Did it end up providing better results in private testnet?

File: narwhal/config/src/committee.rs [hunk 2]
```diff
@@ -133,7 +133,7 @@ impl Display for AuthorityIdentifier {
 impl Committee {
     /// Any committee should be created via the CommitteeBuilder - this is intentionally be marked as
     /// private method.
-    fn new(authorities: BTreeMap<PublicKey, Authority>, epoch: Epoch) -> Self {
+    pub fn new(authorities: BTreeMap<PublicKey, Authority>, epoch: Epoch) -> Self {
```
Comment (dmitri-perelman) [inline narwhal/config/src/committee.rs:136]: Leftover that you forgot to remove?

File: crates/sui-core/src/consensus_adapter.rs [hunk 3]
```diff
@@ -381,24 +385,12 @@ impl ConsensusAdapter {
                 // If we are a low scoring authority, we do not filter out ourselves from the list,
```
Comment (dmitri-perelman) [inline crates/sui-core/src/consensus_adapter.rs:385]: Let's update the comment as well

File: crates/sui-core/src/consensus_adapter.rs [hunk 4]
```diff
@@ -381,24 +385,12 @@ impl ConsensusAdapter {
...
             })
             .collect();
 
         let position = get_position_in_list(self.authority, filtered_positions);
-        (
-            position,
-            mapped_to_low_scoring
-                || (position == 0 && self.authority_is_low_scoring(&self.authority)),
-        )
+        (position, position < initial_position)
```
Comment (dmitri-perelman) [inline crates/sui-core/src/consensus_adapter.rs:393]: nice!

File: narwhal/config/src/committee.rs (hunk 2 above)
Comment (laura-makdah) [reply narwhal/config/src/committee.rs:136]: nice catch

File: narwhal/node/src/primary_node.rs (hunk 1 above)
Comment (laura-makdah) [reply narwhal/node/src/primary_node.rs]: 10k is too much, trying again with 1k

Comment (laura-makdah) [reply narwhal/node/src/primary_node.rs]: [not more stable,](https://mysten.grafana.net/d/FaNbUhene/sui-validators-narwhal?orgId=1&var-Environment=mysten-metrics-internal&var-network=private-testnet&var-validator=All&from=1679629233337&to=1679632205909&showCategory=Thresholds&viewPanel=233) reverting to 300

**Review by dmitri-perelman — APPROVED** [review]

**Pull request merged**